spec/support/currently_undocumented_cops.txt

```diff
@@ -1,3 +1,3 @@
 RuboCop::Cop::Lint::Syntax
-RuboCop::Cop::Migration::DepartmentName
 RuboCop::Cop::Style::ConditionalAssignment
+RuboCop::Cop::Migration::DepartmentName
```

Comment:
It still shows up?

Reply:
That appears to be [part of Rubocop](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/migration/department_name.rb). (Yeah, it confused me too.)